### ISSUE TITLE
Update French translation (strings.xml)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#298]
```
Hi,

Attached is a patch for AndorsTrail/res/values-fr/strings.xml against r220.

It's a major translation update, being more consistent and overall (I believe) 
more satisfying :)
I've also translated the few 6.11-related new strings.
```

Original issue reported on code.google.com by `zorun...@gmail.com` on 20 Feb 2012 at 5:32

Attachments:
- [fr-strings.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-298/comment-0/fr-strings.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the update!
```

Original comment by `sdeva...@gmail.com` on 21 Feb 2012 at 5:27
- Changed state: **Accepted**
- Added labels: **Type-Translation**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
You're welcome :)

Are you ok with the patch or do you need the modified source file?
```

Original comment by `zorun...@gmail.com` on 24 Feb 2012 at 12:20

**Comment by GoogleCodeExporter** [comment]
```
Attached is an update (with lots of new stuff from 6.10 & 6.11) of 
'values-fr/content_itemlist.xml'

I've also attached 'readme-fr.txt', to be put in the same directory, for the 
sake of consistency across French translaters (I don't think anybody does that 
currently)

Something I was wondering: what do you do when you want to change some 
properties of an item (eg. price)? Because properties are copied verbatim from 
the original file to the localized one, without being updated afterwards.
```

Original comment by `zorun...@gmail.com` on 24 Feb 2012 at 4:45

Attachments:
- [content_itemlist.xml](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-298/comment-3/content_itemlist.xml)
- [readme-fr.txt](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-298/comment-3/readme-fr.txt)

**Comment by GoogleCodeExporter** [comment]
```
For some reason, there wasn't any 'content_monsterlist.xml' file for French.

Attached is a first try (about 40% complete), I might finish it some day...
```

Original comment by `zorun...@gmail.com` on 24 Feb 2012 at 10:34

Attachments:
- [content_monsterlist.xml](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-298/comment-4/content_monsterlist.xml)

**Comment by GoogleCodeExporter** [comment]
```
Is there any issue with integrating all these translations?
```

Original comment by `zorun...@gmail.com` on 10 Mar 2012 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r223.

I also added your name to the list of authors for the game, by the "French 
translation by.."-section.
```

Original comment by `oskar.wi...@gmail.com` on 12 Mar 2012 at 9:30
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I'm completed the strings.xml french translation.


```

Original comment by `thon...@gmail.com` on 18 Apr 2012 at 4:19

Attachments:
- [strings.xml](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-298/comment-8/strings.xml)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good (again). The last file from comment 8 is now committed in 
r243.
```

Original comment by `oskar.wi...@gmail.com` on 8 May 2012 at 8:20

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
